### ISSUE TITLE
Embedded additional headwords

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/alternateheadwords#5]
I'm not sure whether you want to consider this topic within this repository.

Many dictionaries have additional headwords 'embedded' within the given entries of the dictionary.

For example,  many dictionaries have prefixed verbs indicated by various prefix sections within the body
of the entry for the verb.

Other dictionaries, like STC, I think, have many compounds indicated only as sub-entries under a
major headword.

So the idea for consideration is to identify these embedded headwords.

**Comment by gasyoun** [comment]
> For example, many dictionaries have prefixed verbs indicated by various prefix sections within the body
> of the entry for the verb.

Yes, sure. Dhaval has requested you to look at PWG, PWK to see what dhatus (`nI` is a good example) take what upasargas (sandhi rules apply in many cases, so it's not just mechanical and @drdhaval2785 code might help). And yes, I think the issue is at home here.

**Comment by drdhaval2785** [comment]
https://github.com/sanskrit-lexicon/alternateheadwords/blob/master/scripts/embedded.py is the script which is used to scrape embedded items out of .txt files.
Right now made to work only for STC and PWG. Generic enough to apply to other dictionaries too.
